thriftcli/thrift_argument_converter.py

```diff
@@ -183,7 +183,7 @@ def _construct_set_arg(self, field_type, value):
 
         """
         elem_type = field_type[field_type.index('<') + 1: field_type.rindex('>')]
-        return frozenset([self._convert_dict_entry_to_arg(elem_type, elem) for elem in value])
+        return [self._convert_dict_entry_to_arg(elem_type, elem) for elem in value]
```

Comment:
This could have consequences. Frozenset is hashable and immutable. List is neither. Are you sure that wont cause problems?